tools/bumper/cnao_repo_commands_test.go

```diff
@@ -42,7 +42,7 @@ var _ = Describe("Testing internal git CNAO Repo", func() {
 		dummyRepo := "dummyRepo"
 		BeforeEach(func() {
 
-			newPr := getFakePrWithTitle("CNAO test-component to 0.0.2", "master")
+			newPr := getFakePrWithTitle("CNAO test-component to 0.0.2", "main")
```

Comment:
currently go-git does not support a repo opened with default branch name other than `master` ([see https://github.com/go-git/go-git/issues/363](https://github.com/go-git/go-git/blob/4ec1753b4e9324d455d3b55060020ce324e6ced2/repository.go#L88))

I suggest we do it manually on our own by setting HEAD to main. please change `newLocalGitRepo` as follows:
```
func setBranchAsHead(repo *git.Repository, branchName string) error {
	h := plumbing.NewSymbolicReference(plumbing.HEAD, plumbing.NewBranchReferenceName(branchName))
	if err := repo.Storer.SetReference(h); err != nil {
		return err
	}
	return nil
}

func newLocalGitRepo(repoDir string, tagCommitMap map[string]string) *gitRepo {
	By(fmt.Sprintf("creating new repository on directory: %s", repoDir))
	repo, err := git.PlainInit(repoDir, false)
	Expect(err).ToNot(HaveOccurred(), "Should succeed cloning git repo")

	setBranchAsHead(repo, "main")

	initializeRepo(repo, repoDir, tagCommitMap)

	return &gitRepo{
		repo:     repo,
		localDir: repoDir,
	}
}
```
I think this should solve your issue